apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -157,6 +171,7 @@ public static function is_active() {
 		if ( true === apply_filters( 'a8c_disable_full_site_editing', false ) ) {
 			return false;
 		}
+		return current_theme_supports( 'full-site-editing' );
```

Comment:
Oh did you get this to work with rest requests?

Reply:
Yup, just needed to hook into `restapi_theme_init`, which fires after the theme files are loaded.

Reply:
Oh, actually, mostly sure this will not work for `/me/sites` as it stands.